UI/ScoreAside/index.js

```diff
@@ -23,7 +33,7 @@ export function ScoreAside({turnNumber, wins, cardsLeft, time}) {
 
             </li>
             <li className={asideStyles.time}>
-               <span className={asideStyles.stat}>{time}</span>
+               <span className={asideStyles.stat}>{timeElapsed}</span>
```

Comment:
Should the time be in a more time looking format?

![image](https://user-images.githubusercontent.com/6362725/102368498-d0846c00-3fb2-11eb-811e-f931e92ad947.png)